Python, doc: Note ephemeral nature of import nodes (#5486)
github/codeql: docs/codeql/codeql-language-guides/using-api-graphs-in-python.rst

➤ `See this in the query console on LGTM.com <https://lgtm.com/query/1876172022264324639/>`__.

> This query selects the API graph node corresponding to the ``re`` module. This node represent the fact that the ``re`` module has been imported rather than a specific place in the program where the import happens. There will, therefore, be at most one result per file, and it will not have a useful location, so you have to click `Show 1 non-source result` in order to see it.

Reviewer: I was confused about the `one result per file` part. I guess it should be `one result per project`? (I also reworded things slightly in my suggestion, which I think is a slight improvement, but nothing major)

```suggestion
This query selects the API graph node corresponding to the ``re`` module. This node represent the fact that the ``re`` module has been imported rather than a specific place in the program where the import happens. Therefore, there will be at most one result per project, and it will not have a useful location, so you have to click `Show 1 non-source result` in order to see it.
```

Author: Thanks, good catch.

> This query selects the API graph node corresponding to the ``re`` module. This node represent the fact that the ``re`` module has been imported rather than a specific place in the program where the import happens. Therefore, there will be at most one result per project, and it will not have a useful location, so you have to click `Show 1 non-source result` in order to see it.

Reviewer 2: Fixing the verb and suggesting two small improvements:
```suggestion
This query selects the API graph node corresponding to the ``re`` module. This node represents the fact that the ``re`` module has been imported rather than a specific location in the program where the import happens. Therefore, there will be at most one result per project, and it will not have a useful location, so you'll have to click `Show 1 non-source result` in order to see it.
```

> To find places in the program where the ``re`` module is referenced, you can use the ``getAUse`` method. The following query selects all references to the ``re`` module in the current database.

Reviewer 2: Feel free to ignore - I just find odd the use of "places in the program", so suggesting an alternative.

```suggestion
To find where the ``re`` module is referenced in the program, you can use the ``getAUse`` method. The following query selects all references to the ``re`` module in the current database.
```
